### DISCUSSION TITLE
GA401QM default AC GPU overclocks

### DISCUSSION THREAD

**Discussion by LoreAPC** [seerge/g-helper#1521] [Q&A]
Hi, is there any owner of the GA401QM model (RTX 3060) with Armoury Crate still on that can tell me the defaul/max OC values (GPU clock and memory clock) of the default Turbo profile?

I'd like to avoid a redownload/reinstall of AC only to get the values...

Thank you very much to anyone answering!

**Comment by seerge** [comment]
@LoreAPC I don't have AC and old G14, but from experience with other devices Turbo usually sets something like +100/+100 for Core/Memory :)

**Comment by LoreAPC** [comment]
I couldn't remember if it was +200/+200 or +100/+100.
My plan was to set these values in the balanced and turbo modes, then apply a core clock limit (set to the max cloxk with no OC) in the balanced plan to effectively have a "default clock + undervolt" balanced plan and a "effective overclock" turbo plan.

This based on my understanding of the "How to undervolt GPU" section of the readme.

Please correct me if I misunderstood something...
